ReactiveXComponent/WebSocket/WebSocketMessageHelper.cs

```diff
@@ -135,13 +134,13 @@ public static WebSocketMessage DeserializeRequest(string request)
                 int firstDotIndex = firstPartBeforeJson.IndexOf('.');
 
                 // Assume it is an input on a custom type of topic if it doesn't have the format  topic = {key}.*.*.* (e.g. output.1_0.engine1.OrderService.CreationFacade)
-                key = firstDotIndex >= 0 ? firstPartBeforeJson.Substring(0, firstDotIndex) : WebSocketCommand.Input;
-                topic = tokensBeforeJson[0];
+                key = firstDotIndex > 0 ? firstPartBeforeJson.Substring(0, firstDotIndex) : WebSocketCommand.Input;

```

Comment:
to clarify; I think step 1 can be done on this review, since it covers the code in it.
step 2 can be done on a future review (that's why i said "eventually").